static/app/components/onboarding/gettingStartedDoc/useSourcePackageRegistries.tsx

```diff
@@ -0,0 +1,45 @@
+import {useEffect} from 'react';
+
+import {Client} from 'sentry/api';
+import {handleXhrErrorResponse} from 'sentry/utils/handleXhrErrorResponse';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+type ReleaseRegistrySdk = Record<
+  string,
+  {
+    canonical: string;
+    main_docs_url: string;
+    name: string;
+    package_url: string;
+    repo_url: string;
+    version: string;
+  }
+>;
+
+// This exists because /extensions/type/search API is not prefixed with
+// /api/0/, but the default API client on the abstract issue form is...
+const API_CLIENT = new Client({baseUrl: '', headers: {}, credentials: 'omit'});
+
+/**
+ * Fetches the release registry list for SDKs
+ */
+export function useSourcePackageRegistries() {
+  const {isLoading, data, error} = useApiQuery<ReleaseRegistrySdk>(
+    ['https://release-registry.services.sentry.io/sdks'],
+    {
+      staleTime: Infinity,
+    },
+    API_CLIENT
+  );
+
+  useEffect(() => {
+    if (error) {
+      handleXhrErrorResponse('Failed to fetch sentry release registry', error);
+    }
+  }, [error]);
```

Comment:
we should handle capture errors in `useApiQuery` hook, but it doesn't seem to do it now